source/docs/articles/drupal.md

```diff
@@ -25,14 +29,14 @@ See [Updating Modules Through Drupal](/docs/articles/drupal/updating-modules-thr
 ## Apache Solr
 [Apache Solr](/docs/articles/sites/apache-solr/) is an open source system for indexing and searching website content. We manage a cluster of fast Solr index servers that communicate with your site via these popular Drupal modules:
 
-​[Apache Solr](https://drupal.org/project/apachesolr) - 7.x-1.4 and 6.x-1.8  
-[Search API Solr](https://drupal.org/project/search_api_solr) - 7.x-1.2
+​[Apache Solr](https://drupal.org/project/apachesolr) - 7.x-1.7 and 6.x-3.0  
+[Search API Solr](https://drupal.org/project/search_api_solr) - 7.x-1.9 and 8.x-1.0
 
 For most sites, the apachesolr module is the easiest to configure and maintain, and includes functionality like facets and other great features.
 
 ## Launch Check
 
-Pantheon provides static site analysis as a service for Drupal 7 sites to make best practice recommendations on site configurations. Go live with confidence using [Launch Check for Drupal](/docs/articles/drupal/launch-check-drupal-performance-and-configuration-analysis/).
+Pantheon provides static site analysis as a service for Drupal sites to make best practice recommendations on site configurations. Go live with confidence using [Launch Check for Drupal](/docs/articles/drupal/launch-check-drupal-performance-and-configuration-analysis/).
```

Comment:
[Site Audit](https://www.drupal.org/project/site_audit) for D8 is in development, only stable release is for D7.